pkg/controller/servicebinding/secret.go

```diff
@@ -49,7 +112,7 @@ func (s *secret) createOrUpdate(payload map[string][]byte, ownerReference metav1
 	resourceClient := s.buildResourceClient()
 
 	logger.Debug("Attempt to create secret...")
-	existingSecret, err := s.get()
+	_, err = s.get()
 	if err != nil {
 		if errors.IsNotFound(err) {
 			_, err := resourceClient.Create(u, metav1.CreateOptions{})
```

Comment:
If `Create` returns an `AlreadyExists` error, it's likely the operation will be retried when it isn't desirable to do so.

Reply:
Is this okay -> 361639e ?